collated/main/A0139916U.md

```diff
@@ -0,0 +1,2151 @@
+            model.removeAliasSymbol(toRemove);
+        } catch (SymbolKeywordNotFoundException e) {
+            e.printStackTrace();
+        }
+        
+        return true;
+    }
+
+    /**
+     * Check if command is an undo command
+     * @return true if the command is an undo operation, false otherwise
+     */
+    @Override
+    public boolean isUndo() {
+        return false;
+    }
+    
+    /**
+     * Check if command is a redo command
+     * @return true if the command is a redo operation, false otherwise
+     */
+    @Override
+    public boolean isRedo(){
+        return false;
+    }
+}
+```
+###### \java\seedu\savvytasker\logic\commands\Command.java
+``` java
+    /**
+     * Checks if a command can perform undo operations
+     * @return true if the command supports undo, false otherwise
+     */
+    public abstract boolean canUndo();
+    
+    /**
+     * Redo the command.
+     * @return true if the operation completed successfully, false otherwise
+     */
+    public abstract boolean redo();
+    
+    /**
+     * Undo the command
+     * @return true if the operation completed successfully, false otherwise
+     */
+    public abstract boolean undo();
+    
+```
+###### \java\seedu\savvytasker\logic\commands\UnaliasCommand.java
+``` java
+package seedu.savvytasker.logic.commands;
+
+import seedu.savvytasker.commons.exceptions.IllegalValueException;
+import seedu.savvytasker.model.alias.AliasSymbol;
+import seedu.savvytasker.model.alias.DuplicateSymbolKeywordException;
+import seedu.savvytasker.model.alias.SymbolKeywordNotFoundException;
+
+/**
+ * Command to remove aliases
+ */
+public class UnaliasCommand extends ModelRequiringCommand {
+
+    public static final String COMMAND_WORD = "unalias";
+
+    public static final String MESSAGE_USAGE = COMMAND_WORD + ": Removes an alias for a command or parameter. "
+            + "Parameters: s/SHORT_KEYWORD\n"
+            + "Example: " + COMMAND_WORD
+            + " s/pjm";
+
+    public static final String MESSAGE_SUCCESS = "Alias removed: %1$s";
+    public static final String MESSAGE_UNREGOGNIZED_ALIAS = "This alias is not in use";
+
+    private final String keyword;
+    private AliasSymbol toUndo;
+    
+    /**
+     * Convenience constructor using raw values.
+     *
+     * @throws IllegalValueException if any of the raw values are invalid
+     */
+    public UnaliasCommand(String keyword) {
+        this.keyword = keyword;
+        this.toUndo = null;
+    }
+
+    @Override
+    public CommandResult execute() {
+        assert model != null;
+        
+        AliasSymbol toRemove = null;
+        for(AliasSymbol symbol : model.getSavvyTasker().getReadOnlyListOfAliasSymbols()) {
+            if (symbol.getKeyword().equals(this.keyword)) {
+                toRemove = symbol;
+                break;
+            }
+        }
+        
+        try {
+            if (toRemove == null)
+                return new CommandResult(MESSAGE_UNREGOGNIZED_ALIAS);
```

Comment:
Use appropriate coding standards for `if` statements (under Conditionals point 10).